pdfobject.js

```diff
@@ -88,9 +88,11 @@
     isIE = function (){ return !!(window.ActiveXObject || "ActiveXObject" in window); };
 
     // Detect desktop Safari
-    // Relying on the Apple pushNotifications api, implemented in 10.9
-    // See https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/NotificationProgrammingGuideForWebsites/PushNotifications/PushNotifications.html
-    isSafariOsx = 'safari' in window;
+    isSafariOsx = (
+        !isIOS &&
+        navigator.vendor && navigator.vendor.indexOf('Apple') !== -1 &&
+        navigator.userAgent && navigator.userAgent.indexOf('Safari') !== -1
+    );
```

Comment:
`'safari' in window` detection does not work for iframes - safari.pushNotifications is only defined on top level window object.